controllers/apply/awards-for-all/lib/contacts.js

```diff
@@ -2,14 +2,19 @@
 const get = require('lodash/fp/get');
 const { safeHtml } = require('common-tags');
 
-function getContactFullName(contactData) {
+function getContactFullName(contactData, html = true) {
```

Comment:
Rather than a boolean could we instead move the HTML to where it's called?

Reply:
Good shout, can you give this another look?

Reply:
Yeah, that's way simpler. Cherrs for that!